dev/ci/12875-herbelin-master+about-print-all-arguments-names.sh

```diff
@@ -0,0 +1,6 @@
+if [ "$CI_PULL_REQUEST" = "12875" ] || [ "$CI_BRANCH" = "master+about-print-all-arguments-names" ]; then
+
+    elpi_CI_REF=coq-master+adapt-coq12875-arguments-pass-name-impargs
+    elpi_CI_GITURL=https://github.com/herbelin/coq-elpi
+
+fi
```

Comment:
overlay needs to be in dev/ci/user-overlays not just dev/ci/

Reply:
oups...